client/src/panels/panel_declarations/results/results_intro_text.yaml

```diff
@@ -0,0 +1,77 @@
+      in **Fall {{planning_year_1_short_second}}**.
+    {{/if}}
+  fr: |
+    {{#if is_gov}}
+      En **{{planning_year_1}}**, **{{fmt_big_int depts_with_dps}}** organisations cherchent à atteindre **{{fmt_big_int dp_results}}**
+      résultats, mesurés par **{{fmt_big_int dp_indicators}}** indicateurs. On fera état du rendement des résultats escomptés à **l'automne {{planning_year_1_short_second}}**.
+    {{else}}
+      Dans son {{gl_tt "plan ministériel" "DP"}} de **{{planning_year_1}}**, {{subj_name subject}} cherche à atteindre **{{fmt_big_int dp_results}}** résultats,
+      mesurés par **{{fmt_big_int dp_indicators}}** indicateurs. On fera état du rendement des résultats escomptés
+      pour **{{fmt_big_int num_crs}}** responsabilités essentielles et **{{fmt_big_int num_programs}}** programmes à **l'automne {{planning_year_1_short_second}}**.
+    {{/if}}
+
+
+drr_summary_text:
+  transform: [handlebars,markdown]
+  en: |
+    {{#if is_gov}}
+      In **{{pa_last_year}}**, **{{fmt_big_int depts_with_drrs}}** organizations sought to achieve **{{fmt_big_int drr_results}}** results.
+    {{else}}
+      In its **{{pa_last_year}}** {{gl_tt "Departmental Results Report" "DRR"}}, {{subj_name subject}} sought to achieve **{{fmt_big_int drr_results}}** results.
+    {{/if}}
+    Progress towards meeting these results was measured using **{{fmt_big_int drr_indicators}}** indicators, and a {{gl_tt "Result Status" "RESULTS_STATUS"}} was assigned to each indicator
+    based on the measured outcome of the indicator's target.
+  fr: |
+    {{#if is_gov}}
+      En **{{pa_last_year}}**, **{{fmt_big_int depts_with_drrs}}** organisations ont cherché à atteindre **{{fmt_big_int drr_results}}** résultats. 
+      Les progrès vers l'atteinte de ces résultats ont été mesurés à l'aide de **{{fmt_big_int drr_indicators}}** indicateurs, et
+      un état du résultat a été associé à chaque indicateur selon la mesure du résultat prévu pour la cible de l'indicateur.
+    {{else}}
+      Dans son {{gl_tt "rapport sur les résultats ministériel" "DRR"}} de **{{pa_last_year}}**, {{subj_name subject}}
+      a cherché à obtenir **{{fmt_big_int drr_results}}** résultats. On a mesuré les progrès vers l'atteinte de ces résultats
+      à l'aide de **{{fmt_big_int drr_indicators}}** indicateurs, et un état du résultat a été associé à chaque indicateur
+      selon la mesure du résultat prévu pour la cible de l'indicateur.
+    {{/if}}
+
+
+reports_links_text:
+  transform: [handlebars,markdown]
+  en: |
+    The most recent {{ext_link "Departmental Plans" dp_url}}
+    and {{ext_link "Departmental Results Reports" drr_url}} can be found on canada.ca.
+  fr: |
+    Vous trouverez les plus récents {{ext_link "plans ministériels" dp_url}} et {{ext_link "rapports sur les résultats ministériels" drr_url}}
+    à canada.ca.
```

Comment:
Maybe also clarify that the data can also be found here? Someone who doesn't know may take this as us telling them to stop wasting their time on InfoBase and follow that link.